### PR TITLE
Add new metadata values

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

File: server/models/dataset_helpers/image_metadata.py [hunk 1]
```diff
@@ -361,7 +361,8 @@ def transform(cls, value):
                 cls._assertEnumerated(value, {
                     'histopathology',
                     'serial imaging showing no change',
-                    'single image expert consensus'})
+                    'single image expert consensus',
+                    'confocal with consensus dermoscopy'})
```
Comment (msmolens) [inline server/models/dataset_helpers/image_metadata.py]: Should the new value be added to the single image upload tool as well?

Comment (brianhelba) [reply server/models/dataset_helpers/image_metadata.py]: Good point.

Comment (brianhelba) [reply server/models/dataset_helpers/image_metadata.py]: Fixed.

**Comment by brianhelba** [comment]
@msmolens Just updated this again. Ready for final review.

**Pull request merged**